litestar/response/base.py

```diff
@@ -113,6 +113,41 @@ def __init__(
         self.is_head_response = is_head_response
         self.encoding = encoding
 
+    @staticmethod
+    def render(content: Any, media_type: str, enc_hook: Serializer, encoding: str) -> bytes:
```

Comment:
Why not move this to `Response` and have `ASGIResponse` accept `str`/`bytes` only?
`Response` could call it in `to_asgi_response`

Reply:
This would be an even stronger separation of concerns, since `ASGIResponse` would then *only* responsible for the sending part, and everything else would be handled externally, which is kinda of what we're going for, right?

Reply:
Thinking about it, it's also better from a user's perspective, since we plan to have them interact with `Response` for the most part, so customisation could happen there more easily.

Reply:
OK good point.